packages/twenty-front/src/modules/ui/layout/page/PageHeader.tsx

```diff
@@ -109,11 +110,12 @@ export const PageHeader = ({
   const isMobile = useIsMobile();
   const theme = useTheme();
   const isNavigationDrawerOpen = useRecoilValue(isNavigationDrawerOpenState);
+  const isSettingsPage = useIsRoute('/settings/profile');
```

Comment:
we should not only check for settings/profile but for all settings route

Reply:
https://stackoverflow.com/questions/61235191/react-router-v6-check-if-path-matches-a-pattern

Reply:
Thanks for the link, I'll have a look